lib/util/init.git.js

```diff
@@ -13,9 +13,10 @@ const git = require('simple-git')();
 const Init = {
     /**
      * check if git repo exists and otherwise create one
-     * @param {Object} [skipInteraction] signals what to insert automatically for things usually asked via wizard
-     * @param {String} skipInteraction.gitRemoteUrl URL of Git remote server
-     * @returns {Promise<{status:String, repoName:String}>} success flag
+     *
+     * @param {object} [skipInteraction] signals what to insert automatically for things usually asked via wizard
+     * @param {string} skipInteraction.gitRemoteUrl URL of Git remote server
+     * @returns {Promise<{status: string, repoName: string}>} success flag
```

Comment:
i wonder if this missing "object"

Reply:
not sure how one would do it, open to suggestions

Reply:
nope, my bad. all good